code/game/objects/items/potions.dm

```diff
@@ -239,18 +239,20 @@
 	return ishuman(user)
 
 /obj/item/potion/zombie/imbibe_effect(mob/living/carbon/human/user)
-	user.become_zombie_after_death = 2
```

Comment:
these are not equivalent, are you sure?

Reply:
become_zombie and make_zombie are different.. perhaps I should change make_zombie into zombify

Reply:
become_zombie and make_zombie are different.. perhaps I should change make_zombie into zombify